src/kirby/components/list/list.component.html

```diff
@@ -1,8 +1,5 @@
-<ion-list kirbyInfiniteScroll
-          (scrollEnd)="onLoadOnDemand()"
-          [disabled]="!isLoadOnDemandEnabled"
-          [class.has-header]="listHeaderTemplate"
-          [class.has-footer]="listFooterTemplate">
+<ion-list kirbyInfiniteScroll (scrollEnd)="onLoadOnDemand()" [disabled]="!isLoadOnDemandEnabled"
```

Comment:
Do we have a defined style rule, on how we position attributes? Inline or on newline. @jakobe or @hafstad any points on this?

Reply:
Don't think so, but in general I think we should keep the formatting and only change it if we decide to...